gotty.go

```diff
@@ -21,33 +22,29 @@ import (
 // If something went wrong reading the terminfo database file, an error is
 // returned.
 func OpenTermInfo(termName string) (*TermInfo, error) {
-	var term *TermInfo
-	var err error
 	// Find the environment variables
-	termloc := os.Getenv("TERMINFO")
-	if len(termloc) == 0 {
+	if termloc := os.Getenv("TERMINFO"); len(termloc) > 0 {
+		path := filepath.Join(termloc, string(termName[0]), termName)
```

Comment:
One more thing; `OpenTermInfo("")` will panic. Given that this function is exported, I suggest to add a check for this

```go
if termName == "" {
    nil, errors.New("No terminfo file(-location) found")
}
```